package.json

```diff
@@ -46,6 +55,36 @@
       "git add"
     ]
   },
+  "nyc": {
```

Comment:
Can this be it's own dotfile, like `.nycrc`? Is that a thing?

Reply:
Sucks that we have to maintain this twice... If somehow the config could be javascript, we could leverage variables between this and `jest.config.js`

Reply:
Good point. I'll move the config to a separate file.

Yeah seems like we got a bunch of options for a config file and actually it could be a javascript file named ``nyc.config.js``